enclave/core/sgx/once.c

```diff
@@ -5,34 +5,39 @@
 #include <openenclave/internal/thread.h>
 #include <openenclave/internal/utils.h>
 
+#define SPINLOCK_NOT_USED 0
```

Comment:
Consider renaming these to
FUNC_NOT_INVOKED,
FUNC_BEING_INVOKED,
FUNC_INVOKED

if that reads better.